src/app.js

```diff
@@ -35,6 +35,7 @@ app.get("nunjucks").addGlobal("getContext", function () {
 });
 
 router.use("/oauth2", require("./app/oauth2"));
+router.use("/", require("./app/address"));
```

Comment:
I'm not sure what these paths will look like once we integrate this with the OAuth flow above - I *think* we're going to want to rename this route - but its a small enough change we can deal with it then

Reply:
I agree, as i was unsure on what the route will be I kept it simple to test for now.